internal/testplanet/node.go

```diff
@@ -4,25 +4,40 @@
 package testplanet
 
 import (
+	"bytes"
 	"context"
+	"fmt"
+	"io"
 
+	"github.com/vivint/infectious"
+	"github.com/zeebo/errs"
 	"go.uber.org/zap"
 	"google.golang.org/grpc"
 
+	"storj.io/storj/internal/memory"
+	"storj.io/storj/pkg/eestream"
 	"storj.io/storj/pkg/identity"
+	"storj.io/storj/pkg/metainfo/kvmetainfo"
 	"storj.io/storj/pkg/overlay"
 	"storj.io/storj/pkg/pb"
 	"storj.io/storj/pkg/pointerdb/pdbclient"
+	"storj.io/storj/pkg/storage/buckets"
+	ecclient "storj.io/storj/pkg/storage/ec"
+	"storj.io/storj/pkg/storage/segments"
+	"storj.io/storj/pkg/storage/streams"
 	"storj.io/storj/pkg/storj"
+	"storj.io/storj/pkg/stream"
 	"storj.io/storj/pkg/transport"
+	"storj.io/storj/satellite"
 )
 
 // Node is a general purpose
 type Node struct {
```

Comment:
Let's rename `Node` into `Uplink` to be clearer, it's not being used for other things anymore